lib/leanplum_api/connection.rb

```diff
@@ -43,7 +42,7 @@ def connection
         connection.request :leanplum_response_validation
         connection.request :json
 
-        connection.response :logger, @logger, bodies: true if LeanplumApi.configuration.api_debug
+        connection.response :logger, LeanplumApi.configuration.logger, bodies: true if LeanplumApi.configuration.api_debug
```

Comment:
minor and not sure if it's important, but there is no fallback here, before it was:
`@logger = options[:logger] || Logger.new(STDERR)` so it could not happen to have nil here

Reply:
not only is it not important, it's actively harmful to use another logger because it won't strip out the private keys etc from the logs, so i made it harder to do on purpose.

Reply:
👍